### PR TITLE
P1 todo and Edit name funtionality

### PR DESCRIPTION
**Pull request by dev-prabhat**
image

image

image

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -83,14 +87,25 @@ body{
     cursor: pointer;
 }
 
+.edit-btn{
+    position: fixed;
+    right: 2%;
+    bottom: 2%;
```
Comment (samyakshah3008) [inline src/styles.css:93]: You can use shorthand property here

**Review by samyakshah3008 — APPROVED** [review]
Clean code and UI 
Good for approval

**Review by archit7448 — APPROVED** [review]
code looks perfect looks good to merge

**Review by sudipkundu999 — APPROVED** [review]
Handle the edge cases :
- I can give space as a name and it works
- Delete only completed todo not all todos
- I can give space for todo and it works.

**Pull request merged**